.github/CODEOWNERS

```diff
@@ -260,7 +250,6 @@ app/models/identifier_index.rb @department-of-veterans-affairs/vsa-debt-resoluti
 app/models/inherited_proofing @department-of-veterans-affairs/vsp-identity
 app/models/inherited_proof_verified_user_account.rb @department-of-veterans-affairs/vsp-identity
 app/models/in_progress_form.rb @department-of-veterans-affairs/vfs-authenticated-experience-backend @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
-app/models/lighthouse_document.rb @department-of-veterans-affairs/lighthouse-banana-peels
```

Comment:
I don't see any of these files and folders within `app/models` being replaced with any top-level ownership. If lighthouse-banana-peels doesn't own them, please replace it with `@department-of-veterans-affairs/backend-review-group`

(The `app/models` lines below this comment look good)